library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/BaseCheckBoxMapperTest.kt

```diff
@@ -0,0 +1,291 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.mapper
+
+import android.graphics.drawable.Drawable
+import android.os.Build
+import android.widget.CheckBox
+import com.datadog.android.sessionreplay.forge.ForgeConfigurator
+import com.datadog.android.sessionreplay.internal.recorder.GlobalBounds
+import com.datadog.android.sessionreplay.internal.recorder.ViewUtils
+import com.datadog.android.sessionreplay.internal.recorder.densityNormalized
+import com.datadog.android.sessionreplay.internal.utils.StringUtils
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.datadog.tools.unit.annotations.TestTargetApi
+import com.datadog.tools.unit.extensions.ApiLevelExtension
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import fr.xgouchet.elmyr.annotation.FloatForgery
+import fr.xgouchet.elmyr.annotation.Forgery
+import fr.xgouchet.elmyr.annotation.IntForgery
+import fr.xgouchet.elmyr.annotation.LongForgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class),
+    ExtendWith(ApiLevelExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(ForgeConfigurator::class)
+internal abstract class BaseCheckBoxMapperTest : BaseWireframeMapperTest() {
+
+    lateinit var testedCheckBoxMapper: CheckBoxMapper
+
+    @Mock
+    lateinit var mockUniqueIdentifierResolver: UniqueIdentifierResolver
+
+    @Mock
+    lateinit var mockTextWireframeMapper: TextWireframeMapper
+
+    @Forgery
+    lateinit var fakeTextWireframes: List<MobileSegment.Wireframe.TextWireframe>
+
+    @LongForgery
+    var fakeGeneratedIdentifier: Long = 0L
+
+    @Mock
+    lateinit var mockViewUtils: ViewUtils
+
+    @Forgery
+    lateinit var fakeViewGlobalBounds: GlobalBounds
+
+    private lateinit var mockCheckBox: CheckBox
+
+    @IntForgery(min = 0, max = 0xffffff)
+    var fakeCurrentTextColor: Int = 0
+
+    @FloatForgery(min = 1f, max = 100f)
+    var fakeTextSize: Float = 1f
+
+    @IntForgery(min = 84, max = 100)
```

Comment:
why 84?

Reply:
is the min height I found in my tests but I will add it from the class constant value to be more visible.